packages/api/src/external/carequality/command/cq-patient-data/update-cq-data.ts

```diff
@@ -14,11 +14,13 @@ export async function updateCQPatientData({
   id,
   cxId,
   cqLinks,
+  invalidateLinks,
   requestLinksDemographics,
 }: {
   id: string;
   cxId: string;
   cqLinks?: CQLink[];
+  invalidateLinks?: CQLink[];
```

Comment:
nit: I think it'd be much easier to read the code if we name this `cqLinksToInvalidate`. It reads better, and relates to the other param, `cqLinks`.